frontend/app/components/advanced-trainings-edit.js

```diff
@@ -6,6 +6,8 @@ import { EKMixin, keyUp } from "ember-keyboard";
 import { observer } from "@ember/object";
 
 export default Component.extend(EKMixin, {
+  intl: service(),
```

Comment:
The codemod I used to convert the components didn't work with those components (probably because of the observers). Thus those components are still completely old school.
Best approach would probably be to manually refactor those components, next time you touch them...